### DISCUSSION TITLE
i have a littel problem will installing

### DISCUSSION THREAD

**Discussion by farid-khelil** [cmgmyr/laravel-messenger#404] [Q&A] [answered]
this message appeare wen i try to run this comand :
php artisan vendor:publish --provider="Cmgmyr\Messenger\MessengerServiceProvider" --tag="config"

   Illuminate\Contracts\Container\BindingResolutionException 

  Target class [blade.compiler] does not exist.

  at vendor\laravel\framework\src\Illuminate\Container\Container.php:906
    902▕
    904▕             $reflector = new ReflectionClass($concrete);
    905▕         } catch (ReflectionException $e) {
  ➜ 906▕             throw new BindingResolutionException("Target class [$concrete] does not exist.", 0, $e);
    907▕         }
    908▕
    909▕         // If the type is not instantiable, the developer is attempting to resolve
    910▕         // an abstract type such as an Interface or Abstract Class and there is

  1   [internal]:0
      Illuminate\Foundation\Application::Illuminate\Foundation\{closure}(Object(Pharaonic\Laravel\Messages\MessagesServiceProvider))

  2   vendor\laravel\framework\src\Illuminate\Container\Container.php:904
      ReflectionException::("Class "blade.compiler" does not exist")

after this comend i cant run php artisan serve or any othe comand from this typ

i am using 
php 8.2.12
laravel 5.1.2
composer 2.7.4
please can you provide me withe solosion for this problem

**Comment by cmgmyr** [comment, marked as answer]
You might have to try to install a different version of the package or upgrade Laravel. Laravel 5.1 isn't supported.
